src/debug/debug.yaml

```diff
@@ -60,3 +60,14 @@
     name: reputations
     schema:
       $ref: '#/components/schemas/Reputations'
+- name: debug_bundler_setMempool
```

Comment:
why is it "set" ? that is, it does both "clear" and "add"
I think it should be "debug_bundler_addUserOps", so that it can be used  to "augment" the mempool.
It should behave as a sequence of sendUserOperation, and just assume validation succeeds, and only addresses touched are those explicitly specified (factory,acct,pm)
This way, the bundler can apply the reputation logic to these added UserOps.